drf_api_logger/admin.py

```diff
@@ -96,11 +96,11 @@ def added_on_time(self, obj):
         added_on_time.short_description = 'Added on'
 
         list_per_page = 20
-        list_display = ('id', 'api', 'method', 'status_code', 'execution_time', 'added_on_time',)
+        list_display = ('id', 'api', 'user_id', 'method', 'status_code', 'execution_time', 'added_on_time',)
         list_filter = ('added_on', 'status_code', 'method',)
```

Comment:
I think it will be better if we add 'user_id' in list_filter too!

Reply:
I'm very sorry, I don't think it's a good idea because it will make django admin dashboard look not good as like of user_id can be very long if there are many users, @vishalanandl177 what's your take on this?